apps/prairielearn/src/pages/administratorQueries/administratorQueries.ts

```diff
@@ -0,0 +1,60 @@
+import { z } from 'zod';
+import express = require('express');
+const router = express.Router();
+import asyncHandler = require('express-async-handler');
+import fsPromises = require('node:fs/promises');
+import path = require('path');
```

Comment:
I believe all these imports allow `import from` instead of `import = require`.

Reply:
![screenshot_2023-10-18_at_2 39 36_pm](https://github.com/PrairieLearn/PrairieLearn/assets/114321556/02c696d9-0daa-4807-bca2-19362f5f9f5d)
Unfortunately, we are unable to use `import from` instead of `import = require`.

Reply:
Try `import * as express from 'express'`

Reply:
That worked. Thanks for the help, @jonatanschroeder !

Resolved in commit 3bbab10.

Reply:
Happy to help. BTW, I believe you missed one more require further down.

Reply:
You are correct. This is now resolved in commit 782f18f.

Also, I had issues with importing the asyncHandler on line 3 and reverted this back to `asyncHandler = require(...`.